models/repo/repo.go

```diff
@@ -168,6 +169,7 @@ type Repository struct {
 	IsFsckEnabled                   bool               `xorm:"NOT NULL DEFAULT true"`
 	CloseIssuesViaCommitInAnyBranch bool               `xorm:"NOT NULL DEFAULT false"`
 	Topics                          []string           `xorm:"TEXT JSON"`
+	HashType                        string             `xorm:"NOT NULL DEFAULT sha1"`
```

Comment:
```suggestion
	HashType                        string             `xorm:"NOT NULL DEFAULT 'sha1'"`
```

is needed to avoid a startup error with postgres